src/components/Comment/docs.md

```diff
@@ -0,0 +1,83 @@
+### `<Comment />`
+
+A `<Comment />` is a combination of `<CommentHeader />` and the comment content. The actions are not part of this component.
+
+```react|noSource,span-4
+<Comment
+  timeago='2h'
+  displayAuthor={{
+    profilePicture: '/static/profilePicture.png',
+    name: 'Paul Ullrich',
```

Comment:
Maybe just use Christoph's real name instead of declaring a liquor dealer a Bundesrat? ;)